### PR TITLE
Add information about Alternate Content Sources

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Alternate Content Sources have been introduced as a feature with foreman 3.5. Adding two modules in the Managing Content document :
- A concept module that describes what ACS is
- A procedure module for adding an ACS

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [X] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5 (Satellite 6.12)
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3 (Satellite 6.11, orcharhino 6.1+)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/con_alternate-content-sources-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[id="Alternate_Content_Sources_Overview_{context}"]
+= Alternate Content Sources Overview
+
+Alternate content sources define alternate paths to download content during synchronization.
+This speeds up the repository synchronization.
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources-overview.adoc:5]: This is not always the case. Maybe reword to something like "depending on your infrastructure" or similar. Or maybe physical distance or network connection?

File: guides/common/modules/con_alternate-content-sources-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+The content itself is synchronized from the alternate content source, and only the metadata is downloaded from the actual repository server.
+Alternate content sources can be set up for {Project}, as well as for {SmartProxy}.
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources-overview.adoc:7]: Please reword to something active, aka. "You can set up ...".

File: guides/common/modules/con_alternate-content-sources-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[id="Alternate_Content_Sources_Overview_{context}"]
+= Alternate Content Sources Overview
+
+Alternate content sources define alternate paths to download content during synchronization.
+This speeds up the repository synchronization.
+The content itself is synchronized from the alternate content source, and only the metadata is downloaded from the actual repository server.
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources-overview.adoc:6]: Maybe "Upstream URL" instead of "actual repository server?

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources* > *Add Source*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:5]:
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
. Click *Add Source*.
```

Or maybe combine "Click add source" with "set the source type ..."

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+. Set the *Source type* as *Custom*.
+. Select the *Content type*.
+. Enter a name in the *Name* field and optionally provide a description in the *Description* field.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:8]: Please split this line. You can use `git grep` to see examples on how we handle entering optional descriptions to various entities.

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Enter the Base URL of the alternate content source.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Manual Authentication* or *Content Authentication* credentials, if these are needed.
+. If SSL verification is required, enable *Verify SSl*, and select the SSL CA certificate.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:13]:
```suggestion
. If SSL verification is required, enable *Verify SSl* and select the SSL CA certificate.
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+. Review details and click *Add*.
+. Refresh the alternate content sources by executing the following command:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:15]: I did not test setting up an ACS yet, but does this mean it always requires an interaction with Foreman Server through CLI?

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+# hammer alternate-content-sources refresh --id _alternate content source_ID_
+----
+
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create --name 'example' --base-url http://local-repo.example.com:port --smart-proxy-ids _{SmartProxy}_ID_
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:29]:
```suggestion
# hammer alternate-content-sources create \
--base-url "_http://local-repo.example.com:port_" \
--name "_My_Alternate_Content_Source_Name_" \
--smart-proxy-ids _{SmartProxy}_ID_
```

Should this example use "https"?

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources refresh --id _alternate content source_ID_
+----
+. Add the {SmartProxies} to which the alternate content source is to be synced:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources update --id _alternate content source_ID_ --smart-proxy-ids _{SmartProxy}_ID_
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:47]: Please also split into multiple lines, see above.

**Review by maximiliankolb — COMMENTED** [review]
Made some comments but have not fully tested the procedure myself yet.

File: guides/common/modules/con_alternate-content-sources-overview.adoc [hunk 10]
```diff
@@ -0,0 +1,7 @@
+[id="Alternate_Content_Sources_Overview_{context}"]
+= Alternate Content Sources Overview
```
[the remark this answers is not in the document]
Comment (AkshayGadhaveRH) [reply guides/common/modules/con_alternate-content-sources-overview.adoc:2]: @maximiliankolb makes sense. I was actually wondering if we have alternate content source as a title for this chapter and then included the different cases (ACS, simplified ACS, and so on) as sub-sections. What would you suggest?

[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/con_alternate-content-sources-overview.adoc:2]: Yeah, sounds like a plan! Maybe ACD helps as an example with its structure: https://docs.theforeman.org/nightly/Deploying_Hosts_AppCentric/index-katello.html

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 7 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:16]: Hey @ianballou can you weigh in on this? Do we need to refresh the ACS through CLI every time?

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 8 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc]: @ianballou could you help with this too?

**Comment by AkshayGadhaveRH** [comment]
@maximiliankolb I was wondering whether we need to reword the ACS definition to something like:

`Alternate Content Source (ACS) points to an alternate location to download content during synchronization.` 
**or**
`Alternate Content Sources (ACS) are used to synchronize content from a location other than the upstream URL.`

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 7 above)
Comment (ianballou) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:16]: ACSs need to be refreshed before they are used. That being said, there are multiple ways to refresh an ACS:

1) Refresh via the UI
2) Refresh via hammer
3) Refresh via the ACS refresh Cron job that runs weekly by default

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 8 above)
Comment (ianballou) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc]: It could be HTTP or HTTPS. We can have help text say `https` just to give a nudge towards being more secure.

**Comment by ianballou** [comment]
> `Alternate Content Sources (ACS) are used to synchronize content from a location other than the upstream URL.`

If you change "synchronize" to "download" I think this is great. There's a sorta difference here because the upstream metadata is still being used to determine what content should be in the final synchronized repository.

**Comment by ianballou** [comment]
Will the simplified and RHUI ACS types be introduced in separate PRs?

**Comment by AkshayGadhaveRH** [comment]
@maximiliankolb right! Using download instead of synchronize makes much more sense. 

Also, yes I'll introduce simplified ACS as a separate chapter in a different PR.

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 8 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc]: @ianballou @maximiliankolb I'll change the example to say `https`. Thanks :)

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 7 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:16]: Hey @ianballou so to refresh via UI, will the user only have to refresh the browser page, right?

File: guides/common/modules/con_alternate-content-sources.adoc [hunk 11]
```diff
@@ -0,0 +1,7 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources.adoc:1]: File name does not match your title/anchor.

File: guides/common/modules/con_alternate-content-sources.adoc [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source, and only the metadata is downloaded from the upstream URL.
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources.adoc:5]:
```suggestion
The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
```

File: guides/common/modules/con_alternate-content-sources.adoc [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+Alternate content source can be used to speed up synchronization if the content is located on the local filesystem or on a nearby network.
+You can set up alternate content sources for {Project}, as well as for {SmartProxy}.
```
Comment (maximiliankolb) [inline guides/common/modules/con_alternate-content-sources.adoc:7]:
```suggestion
You can set up alternate content sources for {Project} and {SmartProxies}.
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create \
+--name 'example' \
+--base-url https://local-repo.example.com:port \
+--smart-proxy-ids _{SmartProxy}_ID_
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:33]:
```suggestion
--base-url "_https://local-repo.example.com:port_" \
--name "_My_ACS_Name_" \
--smart-proxy-ids _{SmartProxy}_ID_
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+----
+# hammer alternate-content-sources refresh --id _alternate content source_ID_
+----
+. Add the {SmartProxies} to which the alternate content source is to be synced:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources update \
+--id _alternate content source_ID_ \
+--smart-proxy-ids _{SmartProxy}_ID_
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:53]:
```suggestion
--id _My_Alternate_Content_Source_ID_ \
--smart-proxy-ids _{SmartProxy}_ID_
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 16]
```diff
@@ -0,0 +1,60 @@
...
+----
+. Refresh the alternate content sources:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources refresh --id _alternate content source_ID_
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:59]:
```suggestion
# hammer alternate-content-sources refresh --id _My_Alternate_Content_Source_ID_
```

**Review by maximiliankolb — COMMENTED** [review]
made some minor suggestions. Please also adjust the procedure to refresh ACS in WebUI.

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 7 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:16]: @maximiliankolb I am checking this further with Ian and will push a new commit including the UI procedure for refreshing ACS.

File: guides/common/assembly_managing-alternate-content-sources.adoc [hunk 17]
```diff
@@ -0,0 +1,3 @@
+include::modules/con_alternate-content-sources.adoc[]
```
Comment (maximiliankolb) [inline guides/common/assembly_managing-alternate-content-sources.adoc:1]:
```suggestion
include::modules/con_managing-alternate-content-sources.adoc[]
```

File: guides/common/modules/con_managing-alternate-content-sources.adoc [hunk 18]
```diff
@@ -0,0 +1,7 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+Alternate content source can be used to speed up synchronization if the content is located on the local filesystem or on a nearby network.
```
Comment (maximiliankolb) [inline guides/common/modules/con_managing-alternate-content-sources.adoc:6]: Please reword to "you can use".

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 19]
```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Custom*.
+. Select the *Content type*.
+. In the *Name* field, enter a name for the alternate content source.
+. In the the *Description* field, provide an description for the ACS.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:9]:
```suggestion
. Optional: In the the *Description* field, provide an description for the ACS.
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Enter the Base URL of the alternate content source.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Manual Authentication* or *Content Authentication* credentials, if these are needed.
+. If SSL verification is required, enable *Verify SSl* and select the SSL CA certificate.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:14]:
```suggestion
. If SSL verification is required, enable *Verify SSL* and select the SSL CA certificate.
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+. Navigate to *Content* > *Alternate Content Sources* > click the vertical ellipsis next to the newly created alternate content source > Select *Refresh*.
+
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create \
+--name "_My_ACS_name_" \
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:25]:
```suggestion
--name "_My_ACS_Name_" \
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+. Review details and click *Add*.
+. Navigate to *Content* > *Alternate Content Sources* > click the vertical ellipsis next to the newly created alternate content source > Select *Refresh*.
+
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create \
```
Comment (vsedmik) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:24]:
```suggestion
# hammer alternate-content-source create \
```
(plus all occurrences bellow)

Also, we need to define the ACS type on creation time via `--alternate-content-source-type` argument. Possible value(s): 'custom', 'simplified', 'rhui'

**Review by vsedmik — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 22 above)
Comment (vsedmik) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:24]: I'm not sure about the scope of this PR - does it refer to *Custom ACS* only or is it rather generic about ACS in general?
In the later case it might be good to mention the ACS types, their purpose and specifics.

Comment (ianballou) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:24]: I think @AkshayGadhaveRH mentioned that is is only custom, and the RHUI + Simplified docs will come in another PR.

Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:24]: Hey @vsedmik I was planning on making this chapter generic. Should I change the content to say that it is about custom ACS?
Also, as @ianballou pointed out, Simplified ACS will be in a separate PR which will follow this one.

File: guides/common/modules/con_managing-alternate-content-sources.adoc [hunk 23]
```diff
@@ -0,0 +1,7 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
```
Comment (maximiliankolb) [inline guides/common/modules/con_managing-alternate-content-sources.adoc:6]:
```suggestion
You can use alternate content sources to speed up synchronization if the content is located on the local filesystem or on a nearby network.
```

File: guides/common/modules/con_managing-alternate-content-sources.adoc [hunk 24]
```diff
@@ -0,0 +1,7 @@
...
+You can set up alternate content sources for {Project} and {SmartProxy}.
```
Comment (maximiliankolb) [inline guides/common/modules/con_managing-alternate-content-sources.adoc:7]:
```suggestion
You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 22 above)
Comment (vsedmik) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:24]: @AkshayGadhaveRH I just asked because the title says _Configuring Alternate Content Sources_, but the procedure describes the _Custom ACS_ creation only. So I felt we were missing the other types. If you are adding them in next PRs then perfect, we just need to update the hammer command.

**Comment by AkshayGadhaveRH** [comment]
Hey @maximiliankolb does this look good to go?

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Can I merge this @ianballou ?

File: guides/common/modules/con_managing-alternate-content-sources.adoc [hunk 25]
```diff
@@ -0,0 +1,7 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+You can use alternate content sources to speed up synchronization if the content is located on the local filesystem or on a nearby network.
+You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
```
Comment (ianballou) [inline guides/common/modules/con_managing-alternate-content-sources.adoc:7]:
```suggestion
You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
Alternate content sources must be "refreshed" before they work.  There is a weekly cron job that refreshes all alternate content sources. Alternate content sources may also be refreshed manually.
```

**Review by ianballou — COMMENTED** [review]
There is just one important detail that I think is missing:

File: guides/common/modules/con_managing-alternate-content-sources.adoc [hunk 26]
```diff
@@ -0,0 +1,10 @@
+[id="Managing_Alternate_Content_Sources_{context}"]
+= Managing Alternate Content Sources
+
+Alternate content sources define alternate paths to download content during synchronization.
+The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
+You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
+You must refresh the alternate content source after creation or after making any changes.
```
Comment (AkshayGadhaveRH) [inline guides/common/modules/con_managing-alternate-content-sources.adoc:8]: @maximiliankolb @ianballou does this look good?

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 27]
```diff
@@ -0,0 +1,55 @@
...
+--name "_My_ACS_Name_" \
+--alternate-content-source-type custom \
+--base-url "_https://local-repo.example.com:port_" \
+--smart-proxy-ids _{SmartProxy}_ID_
+----
+. Check if the newly created alternate content source is listed:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources list
```
Comment (vsedmik) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:34]:
```suggestion
# hammer alternate-content-source list
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 28]
```diff
@@ -0,0 +1,55 @@
...
+----
+. Refresh the alternate content source:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources refresh --id _My_Alternate_Content_Source_ID_
```
Comment (vsedmik) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:40]:
```suggestion
# hammer alternate-content-source refresh --id _My_Alternate_Content_Source_ID_
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 29]
```diff
@@ -0,0 +1,55 @@
...
+----
+. Add the {SmartProxies} to which the alternate content source is to be synced:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources update \
```
Comment (vsedmik) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:46]:
```suggestion
# hammer alternate-content-source update \
```

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc [hunk 30]
```diff
@@ -0,0 +1,55 @@
...
+--id _alternate content source_ID_ \
+--smart-proxy-ids _{SmartProxy}_ID_
+----
+. Refresh the alternate content sources:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources refresh --id _My_Alternate_Content_Source_ID_
```
Comment (vsedmik) [inline guides/common/modules/proc_configuring-alternate-content-sources.adoc:54]:
```suggestion
# hammer alternate-content-source refresh --id _My_Alternate_Content_Source_ID_
```

**Review by vsedmik — CHANGES_REQUESTED** [review]
Please, update also the other occurrences of the hammer command.

**Comment by AkshayGadhaveRH** [comment]
> Please, update also the other occurrences of the hammer command.

@vsedmik done!

**Review by vsedmik — APPROVED** [review]

File: guides/common/modules/con_managing-alternate-content-sources.adoc (hunk 26 above)
Comment (maximiliankolb) [reply guides/common/modules/con_managing-alternate-content-sources.adoc:8]: Works for me!

File: guides/common/modules/proc_configuring-alternate-content-sources.adoc (hunk 15 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-alternate-content-sources.adoc:48]: still open IMHO

File: guides/common/modules/con_managing-alternate-content-sources.adoc (hunk 26 above)
Comment (ianballou) [reply guides/common/modules/con_managing-alternate-content-sources.adoc:8]: Nice!

**Review by ianballou — APPROVED** [review]

**Comment by AkshayGadhaveRH** [comment]
hey @maximiliankolb all changes committed!

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Squashed, merged to master, and cherry-picked: :white_check_mark: 
   2d221203b..cf4655b7e  3.5 -> 3.5
